### PR TITLE
Bump version to 5.4.1-SNAPSHOT

### PR DESCRIPTION
**Pull request by matriv**
Also, renamed `unreleashed.rst` to `5.4.1.rst` and kept only the `Fixes` section.

### CODE REVIEW

File: docs/appendices/release-notes/5.4.1.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.. _version_5.4.1:
+
+=============
+Version 5.4.1
+=============
+
+.. comment Remove the NOTE below and replace with: "Released on 20XX-XX-XX."
```
Comment (matriv) [inline docs/appendices/release-notes/5.4.1.rst:7]: will update our release template to mention this new scheme.

**Review by mfussenegger — APPROVED** [review]
There is also a changes.txt in the project root that symlinked to the unreleased file.  We can remove that


Are there any docs in the dev/docs folder that need an update to reflect this?

**Review by seut — APPROVED** [review]

File: docs/appendices/release-notes/5.4.1.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.. _version_5.4.1:
+
+=============
+Version 5.4.1
```
[the remark this answers is not in the document]
Comment (matriv) [reply docs/appendices/release-notes/5.4.1.rst:4]: You think the `NOTE::` below, which states `In development` is not enough?
Something like:
```
Version 5.4.1 - Unreleased
```
and without the NOTE, or also keep the NOTE ?

**Comment by matriv** [comment]
> There is also a changes.txt in the project root that symlinked to the unreleased file. We can remove that
> 
> Are there any docs in the dev/docs folder that need an update to reflect this?

Thx for catching this! `CHANGES.txt` is indeed used here: https://github.com/crate/crate/blob/master/app/build.gradle#L148

Should we keep it, always pointing to the unreleased (5.4.1.rst in this case) or work on the gradle script to use the latest `.rst` if no version is provided/file is found? or simple throw an error?

File: docs/appendices/release-notes/5.4.1.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (mfussenegger) [reply docs/appendices/release-notes/5.4.1.rst:4]: > So, something like:

Yes

> and without the NOTE, or also keep the NOTE ?

I'd say with,  and maybe extend it to something like "5.4.1 isn't released yet.  These are the release notes for the upcoming release"

**Comment by mfussenegger** [comment]
> Should we keep it, always pointing to the unreleased (5.4.1.rst in this case) or work on the gradle script to use the latest .rst if no version is provided/file is found? or simple throw an error?

Updating the gradle script to use the highest version sounds like the least ongoing effort

**Review by seut — APPROVED** [review]
👍

**Pull request merged**